cmd/etcd-manager/main.go

```diff
@@ -81,6 +81,7 @@ func main() {
 	flag.StringVar(&o.ClusterName, "cluster-name", o.ClusterName, "name of cluster")
 	flag.StringVar(&o.BackupStorePath, "backup-store", o.BackupStorePath, "backup store location")
 	flag.StringVar(&o.BackupInterval, "backup-interval", o.BackupInterval, "interval for periodic backups")
+	flag.StringVar(&o.DiscoveryPollInterval, "discovery-poll-interval", o.DiscoveryPollInterval, "interval for discovery poll")
```

Comment:
How are we going to set that in the context of kOps? I mean, is it worth enabling the setting via ENV vars also, just like backup retentions?

Reply:
we need modify kops apis to make it configurable

Reply:
As those changes need time to go through the versioning cycle I'd recommend also to add the possibility for env vars

Reply:
yeah there are two PRs open that will both add new flags. IMO that should be implemented then both of these. I would like to get these PRs merged first and after that make common solution for envs